runbotics/runbotics-orchestrator-ui/src/contentful/blog-main/cache.ts

```diff
@@ -8,104 +9,84 @@ import {
 import { Language } from '#src-app/translations/translations';
 
 import { getMainPage } from './api';
-
-function getCurrentDateString() {
-    return new Date().toLocaleDateString();
-}
+import { GetBlogMainCache } from './types';
 
 export function getBlogPostsCache(language: Language = 'en') {
-    return contentfulCache[language].get(`posts_${getCurrentDateString()}`) as
-        | BlogPost[]
-        | undefined;
+    return contentfulCache[language].get('posts') as BlogPost[] | undefined;
 }
 
 export function setBlogPostsCache(language: Language, cacheValue: BlogPost[]) {
-    contentfulCache[language].set(
-        `posts_${getCurrentDateString()}`,
-        cacheValue
-    );
+    contentfulCache[language].set('posts', cacheValue);
+}
+
+export function getSinglePostCacheBySlug(language: Language, cacheKey: string) {
+    return contentfulCache[language].get('postsBySlug')[cacheKey] as BlogPost | undefined;
+}
+
+export function getBlogPostsCacheBySlug(language: Language = 'en') {
+    return contentfulCache[language].get('postsBySlug') as BlogPostsBySlug | undefined;
+}
+
+export function setBlogPostsCacheBySlug(language: Language, cacheValue: BlogPostsBySlug) {
+    contentfulCache[language].set('postsBySlug', cacheValue);
```

Comment:
What do you think about making enum for those string values to have them in one place?